redbot/cogs/streams/streamtypes.py

```diff
@@ -329,6 +332,7 @@ def __repr__(self):
 class HitboxStream(Stream):
 
     token_name = None  # This streaming services don't currently require an API key
+    platform_name = "Hitbox"
```

Comment:
The platform is now called Smashcast and the class is named `HitboxStream` for backwards compatibility.
```suggestion
    platform_name = "Smashcast"
```